benchpress/templates/time_wrap.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -e
+
+/usr/bin/time -p "$@"
```

Comment:
Time might not always be in /usr/bin. Maybe just call 'time' and assume its added in the path?

Reply:
`time` is a built-in shell command in zsh and bash. `/usr/bin/time` provides more detailed data. But yes it's true, potentially `time` might not be installed. 

I'll verify with `time` and see if it's enough. Ideally, Benchpress itself should measure the duration.